sumologic/resource_sumologic_slo.go

```diff
@@ -0,0 +1,416 @@
+								"Rolling",
+								"Calendar",
+							}, false),
+						},
+						"target": {
+							Type:         schema.TypeFloat,
+							Required:     true,
+							ValidateFunc: validation.FloatBetween(0, 100),
+						},
+						"timezone": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"size": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"1d", "2d", "3d", "4d", "5d", "6d", "7d", "8d", "9d", "10d", "11d", "12d", "13d", "14d",
+							}, false),
+						},
+					},
+				},
+			},
+			"indicator": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"evaluation_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"Window",
+								"Request",
+							}, false),
+						},
+						"queries": {
+							Type:     schema.TypeList,
+							Required: true,
+							MinItems: 1,
+							MaxItems: 2,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"query_group_type": {
+										Type:     schema.TypeString,
+										Required: true,
+										ValidateFunc: validation.StringInSlice([]string{
+											"Successful", "Unsuccessful", "Total", "Threshold",
+										}, false),
+									},
+									"query_group": {
+										Type:     schema.TypeList,
+										Required: true,
+										Elem:     queryGroupElemSchema,
+									},
+								},
+							},
+						},
+						"query_type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"Logs", "Metrics",
+							}, false),
+						},
+						"threshold": {
+							Type:     schema.TypeFloat,
+							Required: true,
+						},
+						"op": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"LessThan", "GreaterThan", "LessThanOrEqual", "GreaterThanOrEqual",
+							}, false),
+						},
+						"aggregation": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      "Avg",
+							ValidateFunc: validation.StringMatch(aggrRegex, `value must match : `+sloAggregationRegexString),
+						},
+						"size": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringMatch(windowRegex, `value must match : `+sloAggregationRegexString),
+						},
+					},
+				},
+			},
+			"is_mutable": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"is_locked": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"service": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"application": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
```

Comment:
We can skip these two fields - `service` and `application` since UI doesn't expose them yet.